package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "22-testing_controllers",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) Lab 22 - Testing Angular Controllers ======",
+  "main": "karma.conf.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
+    "test": "./node_modules/karma/bin/karma start --single",
+    "test-watch": "./node_modules/karma/bin/karma start"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/npisciotti1/22-testing_controllers.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/npisciotti1/22-testing_controllers/issues"
+  },
+  "homepage": "https://github.com/npisciotti1/22-testing_controllers#readme",
+  "dependencies": {
+    "angular": "^1.6.3",
+    "babel-core": "^6.24.0",
+    "babel-loader": "^6.4.1",
+    "babel-preset-es2015": "^6.24.0",
+    "cowsay-browser": "^1.1.8",
+    "css-loader": "^0.27.3",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "html-webpack-plugin": "^2.28.0",
+    "node-sass": "^4.5.1",
+    "sass-loader": "^6.0.3",
+    "style-loader": "^0.16.0",
+    "webpack": "^1.14.0"
```

Comment:
I don't remember using those dependencies? I was checking other peoples' package.json files and I didn't see them either?